custom/code_types.inc.php

```diff
@@ -204,13 +204,13 @@ function define_external_table(&$results, $index, $table_name, $col_code, $col_d
 $ct_external_options = $externalCodesEvent->getExternalCodeData();
 
 /**
- *  Checks to see if using spanish snomed
+ *  Checks to see if using spanish language snomed
  */
-function isSnomedSpanish()
+function isSnomedSpanishLang()
```

Comment:
hi @luisuriarte , is this function no longer useful?

Reply:
Hi @stephenwaite. Just change the name to be consistent with the statement. It is Spanish speaking countries not only Spain